lib/modules/accountSwitcher.js

```diff
@@ -434,7 +461,28 @@ function _notifySwitchedAccountElsewhere() {
 		message,
 	});
 }
-
+function autoSwitcher(thing) {
+	const loggedInUsername = loggedInUser();
+	if (!loggedInUsername) return;
+	const currentUserIsAuthor = thing.getAuthor() === loggedInUsername;
+	const autoSwitchNotif = _.once(() => {
```

Comment:
Declaring this within the `autoSwitcher` scope makes `_.once` ineffectual, as it would not prevent `showNotification` from being called for `thing`s.